README.md

```diff
@@ -1 +1,14 @@
-# Dx_BAF
\ No newline at end of file
+# Dx_BAF
+
+All script are made tested using Python 3.6.8
+
+## Installation
+To run the BAF scripts we need to create a virtual python environment
+
+### Make virtual python environment
+```bash
+python3 -m venv venv
+source venv/bin/activate
```

Comment:
would it be beneficial to always confirm you’re in the virtual environment by checking the location of your Python by adding this command: "which python"? same for UMCUGenetics/Dx_UPD
OR 
you can also skip the "activation" and instead directly call the executables that are in the virtual environment's bin directory, for example path/to/venv/bin/python something.py

Reply:
I do not fully understand your comment, but the venv is needed for more than only the python executable.
I will add some clarification to the README.